.env.development

```diff
@@ -0,0 +1,13 @@
+# Set local development ENV overrides below:
+
+# ------------------------
+# Local Dev Environment
+#
+# e.g. When running MP locally and Gravity in docker
+# ------------------------
+# GRAVITY_API_BASE=http://localhost:3000/api/v1
+# GRAVITY_GRAPHQL_ENDPOINT=http://localhost:3000/api
+# GRAVITY_API_URL=http://localhost:3000
+# GRAVITY_ID=metaphysics-client-id
+# GRAVITY_SECRET=metaphysics-client-secret
+# MEMCACHED_URL=localhost:11211
```

Comment:
I'm wondering if it's a bit too early to push these changes. Same with the port changes as I think there is some more flushing out of the approach we want to take. E.g. proving out if we can leverage the reverse proxy auto-discovery. 

If we think pushing these now will help with the development, I'm open, but I think we just need to be realistic that these will likely change as we solidify our approach and that could cause a bit of confusion.

Reply:
Also, should these be uncommented? I think only `hokusai dev` relies on this file, in which case we want these to be loaded. Having the dev uncomment these seems like an extra step.

Reply:
[Removed the .env.development](https://github.com/artsy/metaphysics/pull/5341/commits/c7163bf164a3e9feae452f213785c541a1e89f7a) file until the approach is more flushed out. Started discussion [here](https://artsy.slack.com/archives/CA8SANW3W/p1697210781648959).

Reply:
After discussing, re-added [.env.development](https://github.com/artsy/metaphysics/pull/5341/commits/245a34d27fd1803dd2b18fd5472f91ac8f37839e) with only `localhost` vars to stay consistent with established hokusai development patterns.